PM2 startup setup: no sudo password retry on failure

## Bug

During `zylos init` on macOS, the PM2 boot auto-start step prompts for sudo password. If the password is entered incorrectly, the setup fails immediately with no retry opportunity.

**Reported by:** Howard (Mac Mini M4, macOS)

**Error:**
```
Password:
  ⚠ PM2 boot auto-start: sudo command failed: [PM2][ERROR] Failure when trying to write startup script
    Fix manually: sudo env PATH=$PATH:/Users/howard_mini/.nvm/versions/node/v24.14.0/bin /Users/howard_mini/.nvm/versions/node/v24.14.0/lib/node_modules/pm2/bin/pm2 startup launchd -u howard_mini --hp /Users/howard_mini
```

## Problem

1. User is prompted for sudo password during PM2 startup configuration
2. If the password is wrong, the command fails immediately
3. No retry is offered — the init proceeds with PM2 startup not configured
4. User has to manually run the fix command shown in the error

## Expected Behavior

- Retry sudo password prompt on failure (e.g., up to 3 attempts, same as standard sudo behavior)
- Or at minimum, clearly indicate the password was likely incorrect and prompt the user to try again

--- a/cli/commands/init.js
+++ b/cli/commands/init.js
@@ -1091,10 +1091,10 @@ function setupPm2Startup() {
     if (sudoResult.status === 0) {
       console.log(`  ${success('PM2 boot auto-start configured')}`);
     } else {
-      console.log(`  ${warn('PM2 boot auto-start: sudo password incorrect or command failed')}`);
-      console.log(`    ${dim('This is optional — Zylos works fine without it.')}`);
-      console.log(`    ${dim('To enable auto-start later, run:')}`);
-      console.log(`    ${dim(sudoMatch[1])}`);
+      console.log(`  ${warn('PM2 boot auto-start: sudo password incorrect')}`);
+      console.log(`    This is optional — Zylos works fine without it.`);
+      console.log(`    ${cyan('To enable auto-start later, run:')}`);
+      console.log(`      ${bold(sudoMatch[1])}`);
     }
     return;
   }

